compiler/config/tests.config

```
@@ -15,20 +15,30 @@ bin    = ./scripts/check-cct
 okdirs = !CCT/success
 kodirs = !CCT/fail
 
-[test-all]
-okdirs = !examples !tests/success
-kodirs = tests/fail tests/fail/annotation
-exclude = examples/arm_m4
+[test-x86-64-ATT]
+bin     = ./scripts/check-x86-64
+args    = -ATT
+okdirs  = examples/**/x86-64 tests/success/**/x86-64
+kodirs  = tests/fail/**/x86-64
+exclude = tests/fail/warning/x86-64
 
-[test-print]
+[test-x86-64-Intel]
+bin = ./scripts/check-x86-64
+args = -intel
+okdirs = examples/**/x86-64 tests/success/**/x86-64
+kodirs = tests/fail/**/x86-64
+exclude = tests/fail/warning/x86-64
+
+[test-x86-64-print]
 bin = ./scripts/parse-print-parse
-okdirs = tests/success
+okdirs = tests/success/x86-64
 
-[test-extraction]
+[test-x86-64-extraction]
 bin = ./scripts/extract-and-check
-okdirs = !examples !tests/success
-exclude = tests/success/noextract examples/arm_m4
+okdirs = examples/**/x86-64 tests/success/**/x86-64
+exclude = tests/success/noextract
```

```suggestion
exclude = !tests/success/noextract
```

The subdirectories of `noextract/` should be included too.

---

That's right. All excludes in these test scenarios should be recursive, maybe? (There's just one other directory we exclude, `tests/fail/warning`; it doesn't have subdirectories, but well, we don't want to test warnings for anything we may add in future.)

---

I meant if I should add bangs to the other three excludes in this file, not to change the script.

---

What you propose makes sense.

---

Done.